benchmark/how-to-run-sysbench.md

```diff
@@ -128,66 +116,74 @@ db-driver=mysql
 
 ### Data import
 
-Create the `sbtest` database using the following statement in the MySQL:
+Before importing the data, it is necessary to make some settings on TiDB. Execute the following command in MySQL client:
+
+```sql
+set global tidb_disable_txn_auto_retry = off;
+```
+
+Then exit the client. TiDB uses an optimistic transaction modal that rolls back transactions when a concurrency conflict is found. Setting `tidb_disable_txn_auto_retry` to `off` will turn on the automatic retry mechanism after a transaction conflict, which will try to avoid Sysbench quit caused by the error reports of transaction conflicts.
```

Comment:
```suggestion
Then exit the client. TiDB uses an optimistic transaction model that rolls back transactions when a concurrency conflict is found. Setting `tidb_disable_txn_auto_retry` to `off` turns on the automatic retry mechanism after meeting a transaction conflict, which can prevent Sysbench from quitting because of the transaction conflict error.
```
In a technical document, use present tense rather than future tense; in particular, try to avoid using "will" where possible. Ref: https://developers.google.cn/style/tense